Makefile

```diff
@@ -276,7 +308,12 @@ golangci-lint: ## Run golangci-lint on codebase
 	$(GOBIN)/golangci-lint run ./...
 ifeq ($(BUILD_ENTERPRISE_READY),true)
   ifneq ($(MM_NO_ENTERPRISE_LINT),true)
-	$(GOBIN)/golangci-lint run ../enterprise/...
+		$(GOBIN)/golangci-lint run ../enterprise/...
+  endif
+endif
+ifeq ($(BUILD_BOARDS),true)
+  ifneq ($(MM_NO_BOARDS_LINT),true)
+		$(GOBIN)/golangci-lint run $(BUILD_BOARDS_DIR)/server/...
```

Comment:
mind you, this would mean golangci-lint would run with the server repo's rules, not: https://github.com/mattermost/focalboard/blob/main/server/.golangci.yml. But looking at that, it seems much more exhaustive so it should be fine. We should also remove that file to avoid confusion.